### ISSUE TITLE
Build failure: nodePackages.bash-language-server

### ISSUE THREAD

**Issue by calebstewart** [NixOS/nixpkgs#319871]
### Steps To Reproduce

1. `nix-shell -p nodePackages.bash-language-server`

### Build log

```
error:
       … while calling the 'derivationStrict' builtin

         at /builtin/derivation.nix:9:12: (source not available)

       … while evaluating derivation 'shell'
         whose name attribute is located at /nix/store/qqwr649pc0qprc9lw2fmdsi1km6p7q2h-source/pkgs/stdenv/generic/make-derivation.nix:331:7

       … while evaluating attribute 'buildInputs' of derivation 'shell'

         at /nix/store/qqwr649pc0qprc9lw2fmdsi1km6p7q2h-source/pkgs/stdenv/generic/make-derivation.nix:378:7:

          377|       depsHostHost                = elemAt (elemAt dependencies 1) 0;
          378|       buildInputs                 = elemAt (elemAt dependencies 1) 1;
             |       ^
          379|       depsTargetTarget            = elemAt (elemAt dependencies 2) 0;

       error: Alias bash-language-server is still in node-packages.nix
```

### Additional context

This seems to have been caused by #317954 although I'm not familiar with the package, so I could be wrong. It is causing `nixvim` to fail as a result.

### Notify maintainers

I don't quite understand the organization of the `pkgs/development/node-packages` structure, so I'm not sure who the appropriate maintainer is. As a fallback, @NyCodeGHG and @doronbehar were involved in the above linked PR, so they may be of help here.

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
➜ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.32, NixOS, 24.11 (Vicuna), 24.11.20240613.e9ee548`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.2`
 - nixpkgs: `/nix/store/qqwr649pc0qprc9lw2fmdsi1km6p7q2h-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by NyCodeGHG** [comment]
sorry for the breakage, i opened #319882 to fix this

**Comment by Ate329** [comment]
Had the same issue. But I think it's fixed in PR #319882 as mentioned
You can track the progress of the PR [here](https://nixpk.gs/pr-tracker.html?pr=319882), apparently it hasn't been merged into nixpkgs-unstable yet.
